src/main/resources/db/changelog/changes/create-transaction-table.xml

```diff
@@ -49,7 +49,7 @@
         <constraints nullable="false"/>
       </column>
       <column name="item_title" type="text">
-        <constraints nullable="false"/>
+        <constraints nullable="true"/>
```

Comment:
I think another database change log will be needed. As this module has been deployed already in test environment.